app/styles/views/_nav.scss

```diff
@@ -17,15 +17,29 @@
 	cursor: pointer;
 
 	width: 100%;
-	padding-top: 20px;
-	padding-bottom: 20px;
+	padding-top: 18px;
+	padding-bottom: 18px;
 	text-align: center;
 }
 
+.cr-side-nav__item:hover {
+	background-color:$link-hover-color;
+}
+
 .cr-side-nav__item > img {
 	opacity: 0.4;
 
 }
+
+.cr-side-nav__item:hover > img{
+	opacity: 1.0;
+}
+
+.cr-side-nav__item--active  {
```

Comment:
can we make that 3px solid ... so the formal syntax is in order (i.e. line-width line-style color)?
https://developer.mozilla.org/en-US/docs/Web/CSS/border#formal_syntax for reference

other than that, looks good!